packages/react/context-menu/src/ContextMenu.tsx

```diff
@@ -134,15 +120,17 @@ const ContextMenuTrigger = React.forwardRef<ContextMenuTriggerElement, ContextMe
             // clearing the long press here because some platforms already support
             // long press to trigger a `contextmenu` event
             clearLongPress();
-            event.preventDefault();
             handleOpen(event);
+            event.preventDefault();
           })}
           onPointerDown={composeEventHandlers(
             props.onPointerDown,
             whenTouchOrPen((event) => {
               // clear the long press here in case there's multiple touch points
               clearLongPress();
-              longPressTimerRef.current = window.setTimeout(() => handleOpen(event), 700);
+              longPressTimerRef.current = window.setTimeout(() => {
```

Comment:
nit: was there a reason for this change?

Reply:
Formatting hangover from a previous change I think